frontend/src/components/FragmentComment.js

```diff
@@ -0,0 +1,50 @@
+import '../styles/FragmentComment.css';
+import React from 'react';
+
+function FragmentComment({ children }) {
+  try {
+    const citationRegex = /^\[.*?\]\s*\n(.*)$/m;
+    if (citationRegex.test(children)) {
+      let [citation, commentaire] = children[0].split(/\n/);
```

Comment:
`commentaire`: Please name variables with english term.

Reply:
Thanks for the advice, this is now fixed in the last commit.